releases/note-source/v15_0.rst

```diff
@@ -0,0 +1,108 @@
+
+Performance improvements in coaddition
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now all coaddition algorithms have significantly reduced memory footprints.
+
+.. _release-v15-0-artifacts:
+
+Significantly improved artifact rejection in coaddition
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Coaddition algorithms that do artifact clipping can now handle artifacts that overlap from epoch to epoch. ``SafeClipAssembleCoaddTask`` and ``CompareWarpAssembleCoaddTask`` are the two examples.
+
+.. _release-v15-0-skysub:
+
+Full focal plane sky subtraction
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are now tasks to create and apply models of the sky that extend over the entire field of view. View notes in `the LSST Community form post <https://community.lsst.org/t/sky-subtraction/2415>`_.
+
+.. _release-v15-0-wcs:
+
+Replace all ``Wcs`` classes with the AST backed ``SkyWcs``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The last release introduced a transform system backed by the `AST <https://arxiv.org/abs/1602.06681>`_ package. Since that release the stack has been converted to using that system in all contexts where a world coordinate system is required.
+
+.. _release-v15-0-firefly:
+
+Plotting frontend for Firefly
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This release includes, for the first time, the package that allows the LSST plotting abstraction layer to plot directly in the Science Portal plotting tool, Firefly. View ``display_firefly`` on `GitHub <https://github.com/lsst/display_firefly>`_.
+
+.. _release-v15-0-compression:
+
+Lossless compression on by default
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Lossless compression is turned on by default when persisting any image-like data product. Read the `decision <https://jira.lsstcorp.org/browse/RFC-378>`_ and view the notes in `this LSST Community forum post <https://community.lsst.org/t/lossless-fits-compression-enabled/2410>`_ and links therein.
+
+.. _release-v15-0-cmdline:
+
+Changes to command-line tasks
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Command-line tasks now handle clobbering of versions, data, and configs in a more intuitive way. E.g. output repositories are now expected to differ from the input repository.  This eliminates the need to explicitly turn on clobbering when making multiple runs to different outputs (reruns) when using the same inputs. Additional details are in an `LSST Community forum post <https://community.lsst.org/t/changes-to-command-line-task-behavior/2408>`_.
+
+.. _release-v15-0-prereqs:
+
+Updated pre-requisites
+^^^^^^^^^^^^^^^^^^^^^^
+
+:ref:`Pre-requisites <python-deps>` for installing the science pipelines have been updated. Of note are that ``numpy 1.13``, ``astropy 2.0`` and ``matplotlib 2.0`` are all required. The baseline version of ``Python`` is now ``Python 3.6``.  See the announcement in `this post on the LSST Community forum <https://community.lsst.org/t/dm-python-and-associated-packages-version-baseline-change/2251>`_.
```

Comment:
Capitalize “Science Pipelines”.  Also, I'd write “NumPy 1.13” rather than “`numpy 1.13`”, but that's just style I guess.